onos-umbrella/Chart.yaml

```diff
@@ -17,6 +17,11 @@ maintainers:
   - name: ONOS Support
     email: support@opennetworking.org
 dependencies:
+  - name: ca-issuer-charts
+    condition: import.ca-issuer.enabled
+    #intel-innersource/frameworks.edge.one-intel-edge.maestro-app.roc.rocaas-charts/ca-issuer-charts
```

Comment:
We should remove the intel inner source thing

Reply:
done